jetstream_pt/engine.py

```diff
@@ -188,12 +190,12 @@ def _call_model_prefill(self, weights, tokens, input_indexes):
     mask = jnp.triu(mask, k=1)
     args = (tokens, input_indexes, caches, mask)
 
-    paramst, argst = torch_xla2.tensor.wrap((weights, args))
+    paramst, argst = from_jax((weights, args))
     with self._lock:
-      with torch_xla2.tensor.XLADispatchMode():
+      with torch_xla2.default_env():
```

Comment:
For Pytorch user, DispatchMode is more descriptive than default env? And we also need some comments here why we need TorchDispatchMode here.

Reply:
done